scripts/build.d/pyside6

```diff
@@ -99,11 +99,20 @@ EOF
     exit
   fi
 
-  buildcmd install.log ${PYTHON} setup.py install \
-    --qtpaths=${QTPATHS} \
-    --ignore-git \
-    --reuse-build
+  install_cmd=("${PYTHON}")
+  install_cmd+=("setup.py")
+  install_cmd+=("install")
 
+  if [ $(uname) == Linux ]; then
```

Comment:
It's probably good to leave a code remark pointing to this PR or summarizing the OOM induced with ninja on (Arch) Linux.